### PR TITLE
Login message

### PR DESCRIPTION
**Pull request by GaryDaviesGaiaResources**

### CODE REVIEW

**Comment by GaryDaviesGaiaResources** [comment]
uses the function formatAPIError

File: src/pages/login/login.ts [hunk 1]
```diff
@@ -33,20 +36,25 @@ export class LoginPage {
...
             },
-            (error) => this.alertController.create({
-                title: 'Login Problem',
-                subTitle: error.msg,
-                buttons: ['Ok']
-            }).present()
+            (error) => {
+                this.showSpinner = false;
+                const apiResponse = formatAPIError(error) as ApiResponse;
+                from(apiResponse.non_field_errors).pipe(mergeMap(message => this.alertController.create({
```
Comment (tony-gaia) [inline src/pages/login/login.ts]: I like that you're getting your observable on, but I'm not really sure this is the place to use them - a normal loop or javascript array map would probably suffice.

Comment (tony-gaia) [reply src/pages/login/login.ts]: Also I think formatting all errors and displaying them in a single alert would be better than showing successive alerts.

Comment (GaryDaviesGaiaResources) [reply src/pages/login/login.ts]: changed it to display the first error

File: src/pages/login/login.ts [hunk 2]
```diff
@@ -33,20 +36,25 @@ export class LoginPage {
...
-                title: 'Login Problem',
-                subTitle: error.msg,
-                buttons: ['Ok']
-            }).present()
+            (error) => {
+                this.showSpinner = false;
+                const apiResponse = formatAPIError(error) as ApiResponse;
+                this.alertController.create({
+                    title: 'Login Problem',
+                    subTitle: apiResponse.non_field_errors[0],
```
Comment (serge-gaia) [inline src/pages/login/login.ts]: Suggestion: maybe we can show all the errors in one string separated by new line.
```
subTitle: apiResponse.non_field_errors.join('\n')
```
Will also work if there's only one error. 
It depends if the alert understand new line. If it is html based:
```
subTitle: apiResponse.non_field_errors.join('<br>')
```

**Comment by serge-gaia** [comment]
I have merged but I think we can explore the multi line alert and see how it renders.

File: src/pages/login/login.ts (hunk 2 above)
Comment (serge-gaia) [reply src/pages/login/login.ts]: How does it look like with newline `('\n')`. If it doesn't work use slash `('/')` ?

**Pull request merged**